include/fluent-bit/flb_config.h

```diff
@@ -34,6 +34,9 @@
 #define FLB_CONFIG_FLUSH_SECS   5
 #define FLB_CONFIG_HTTP_LISTEN  "0.0.0.0"
 #define FLB_CONFIG_HTTP_PORT    "2020"
+#define ERROR_COUNTS_DEFAULT 5
+#define RETRY_FAILURE_COUNTS_DEFAULT 5
```

Comment:
nit: Adding "HC" or "HEALTH" or "UNHEALTHY" at the beginning of these names would be ideal since this is the `flb_config.h` which is used for many things